storybook/qmlTests/tests/tst_MintTokensSettingsPanel.qml

```diff
@@ -0,0 +1,96 @@
+                                                            root)
+        }
+
+        function test_mintTokensIntroPages() {
+
+            const introPanel = findChild(mintTokensSettingsPanel, "introPanel")
+            const infoBoxPanel = findChild(mintTokensSettingsPanel,
+                                           "infoBoxPanel")
+            const infoBoxPanelButton = findChild(mintTokensSettingsPanel,
+                                                 "statusInfoBoxPanelButton")
+
+            waitForRendering(mintTokensSettingsPanel)
+            compare(introPanel.title, "Community tokens")
+            compare(introPanel.subtitle,
+                    "You can mint custom tokens and import tokens for your community")
+            compare(JSON.stringify(introPanel.checkersModel), JSON.stringify(
+                        ["Create remotely destructible soulbound tokens for admin permissions", "Reward individual members with custom tokens for their contribution", "Mint tokens for use with community and channel permissions"]))
+
+            compare(infoBoxPanel.title, "Get started")
+            compare(infoBoxPanel.text,
+                    "In order to Mint, Import and Airdrop community tokens, you first need to mint your Owner token which will give you permissions to access the token management features for your community.")
+            compare(infoBoxPanel.buttonText, "Mint Owner token")
+            compare(infoBoxPanel.buttonVisible, true)
+
+            mouseClick(infoBoxPanelButton)
+
+            waitForRendering(mintTokensSettingsPanel)
+            waitForItemPolished(mintTokensSettingsPanel)
+
+            tryCompare(mintTokensSettingsPanel.currentItem, "objectName",
+                       "ownerTokenPage")
+
+            var settingsPage = findChild(mintTokensSettingsPanel.currentItem,
+                                         "welcomeView")
+
+
+            compare(htmlToPlainText(((findChild(settingsPage,
+                                                "introPanelText")).text).replace(/’/g, "'")),
+                    "Your Owner token will give you permissions to access the token management features for your community. This token is very important - only one will ever exist, and if this token gets lost then access to the permissions it enables for your community will be lost forever as well.
+                        Minting your Owner token also automatically mints your community's TokenMaster token.  You can airdrop your community's TokenMaster token to anybody you wish to grant both Admin permissions and permission to access your community's token management functions to.
+                        Only the hodler of the Owner token can airdrop TokenMaster tokens. TokenMaster tokens are soulbound (meaning they can't be transferred), and you (the hodler of the Owner token) can remotely destruct a TokenMaster token at any time, to revoke TokenMaster permissions from any individual.")
```

Comment:
Do we need to put the full text here? Maybe we could just check if it begins with the proper phrase? Having full string here doesn't help probably. We stil have not guarantee it's displayed properly at least. Btw. I would add check if the component is visible.